packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -305,16 +305,30 @@ class SliverChildListDelegate extends SliverChildDelegate {
   }
 }
 
+/// A base class for sliver that have [KeepAlive] children.
+abstract class SliverWithKeepAliveWidget extends RenderObjectWidget {
+  /// Initializes fields for subclasses.
+  const SliverWithKeepAliveWidget({
+    Key key,
+  }) : super(key : key);
+
+  @override
+  SliverWithKeepAliveElementMixin createElement();
```

Comment:
Do we need this?  Or `SliverWithKeepAliveElementMixin`?  It doesn't  look like this really does anything different than if we just kept the super.createElement (returning a `RenderObjectElement`).

Reply:
I think you are right, the `SliverWithKeepAliveElementMixin` is not necessary.